Week 1/Weekly Practice Problems (Week 1)/2469: Convert the Temperature/Bryanna.cpp

```diff
@@ -0,0 +1,9 @@
+class Solution {
+public:
+    vector<double> convertTemperature(double celsius) {
+        vector<double> vect;
+        vect.push_back(celsius + 273.15);
+        vect.push_back(celsius * 1.80 + 32.00);
+        return vect;
```

Comment:
Nice, this works! Though we can make the solution shorter! 
```suggestion
        return {celsius + 273.15, celsius * 1.80 + 32.00};
```